circleci/templates/maven/config_template-no-docker-v1.yml

```diff
@@ -109,13 +109,49 @@ jobs:
       - image: circleci/ruby:2.4.1
     steps:
       - run: echo "*PLACEHOLDER* pretending to execute integration testing phase!"
-  Deploy Artifacts: # Deploy artifacts to Nexus or Maven Central
-    docker:
-      - image: circleci/ruby:2.4.1
-    steps:
-      - run: echo "*PLACEHOLDER* pretending to deploy build artifacts to Maven Central!"
 # END: UNDER CONSTRUCTION AREA ---------------------------------------
 
+  Tag Release:
+    docker: *image
+    steps:
+      # Tag Release creates a Git tag with given name and pushes it to the given GitHub repo
+      # This job is governed by the following environment variables:
+      # $RELEASE - Set to true to perform a release, otherwise this step is skipped
```

Comment:
Move...maybe simplify?  Environment variable call outs to top of yml.

Reply:
👍